tests/baselines/reference/contextualTypingWithFixedTypeParameters1.errors.txt

```diff
@@ -10,8 +9,7 @@ tests/cases/compiler/contextualTypingWithFixedTypeParameters1.ts(3,32): error TS
                          ~~~
 !!! error TS2339: Property 'foo' does not exist on type 'string'.
     var r9 = f10('', () => (a => a.foo), 1); // error
-             ~~~
-!!! error TS2453: The type argument for type parameter 'T' cannot be inferred from the usage. Consider specifying the type arguments explicitly.
-!!! error TS2453:   Type argument candidate 'string' is not a valid type argument because it is not a supertype of candidate 'T'.
                                    ~~~
-!!! error TS2339: Property 'foo' does not exist on type 'T'.
\ No newline at end of file
+!!! error TS2339: Property 'foo' does not exist on type 'string'.
+                                         ~
+!!! error TS2345: Argument of type 'number' is not assignable to parameter of type 'string'.
```

Comment:
We don't consider the outer arrow function to be context sensitive (it has no contextually typed parameters), but the contextual typing of the inner arrow function causes T to be fixed at its current set of inferences (string) during the initial pass. The later inference of number is ignored because T is already fixed.

The issue isn't related to this PR and I'm not sure that we want to fix it--nor am I sure how.